src/Juno/JunoClient.test.ts

```diff
@@ -131,8 +134,16 @@ describe("GitHub", () => {
 
 describe("Gallery", () => {
   const junoClient = new JunoClient(ko.observable<DatabaseAccount>(sampleDatabaseAccount));
+  const originalUserContext = userContext;
+
+  beforeEach(() => {
+    updateUserContext({
+      subscriptionId: sampleSubscriptionId
+    });
+  });
 
   afterEach(() => {
+    updateUserContext(originalUserContext);

```

Comment:
Does this need to get reset every time? Or can you just do it once in a beforeAll()

Reply:
Makes sense. Updated.